.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+# Description
+Fixes #
+
+# Checklist
+- [ ] Updated HISTORY.rst (if needed)
```

Comment:
Since we have a convention of when to update the `HISTORY` (when we have a change that affects the user's experience of the application), I wonder if it'd be useful to note that here.   You could imagine the PR submitted yesterday wondering what 'if needed' is, and different projects may have different approaches.